docs/user_interface.rst

```diff
@@ -2,15 +2,15 @@
 
 .. _user-interface:
 
-User Interface
-==============
-
-.. _user-interface-workspace:
+**************
+Utility Panels
```

Comment:
I don't think this should be called "Utility Panels", which is too specific.

"User Interface" is probably too general.

How about "Project Window" or "Main Window"? I'm open to other ideas.